src/commands/numerics.js

```diff
@@ -147,6 +147,7 @@ module.exports = {
     '484': 'ERR_ISCHANSERVICE',
     '485': 'ERR_ISREALSERVICE',
     '491': 'ERR_NOOPERHOST',
+    '569': 'RPL_WHOISASN',
```

Comment:
Could you include the link to the insp module above this new numeric and a note saying that it's insp specific?